config/zones/MY.yaml

```diff
@@ -0,0 +1,10 @@
+contributors:
+  - kruschk
+  - systemcatch
+  - nessie2013
```

Comment:
Technically not needed since the build script will aggregate the contributors from the subzones to the parent zone, but why don't you add yourself as well ;)